### PR TITLE
docs(operators): add documentation for partition

### PR DESCRIPTION
**Pull request by hardikpthv**
Closes: #120

### CODE REVIEW

File: src/operator-docs/transformation/partition.ts [hunk 1]
```diff
@@ -1,6 +1,69 @@
 import { OperatorDoc } from '../operator.model';
 
 export const partition: OperatorDoc = {
-  'name': 'partition',
-  'operatorType': 'transformation'
+  name: 'partition',
+  operatorType: 'transformation',
+  signature:
+    'public partition(predicate: function(value: T, index: number): boolean, thisArg: any): [Observable<T>, Observable<T>',
```
Comment (JWO719) [inline src/operator-docs/transformation/partition.ts]: here is a little syntax error. At the end, there is a missing ]

Comment (hardikpthv) [reply src/operator-docs/transformation/partition.ts]: hmm i see...

**Comment by ashwin-sureshkumar** [comment]
@hardikpthv - Could please rebase with master

**Comment by ladyleet** [comment]
thank you for merging this @ashwin-sureshkumar! and thx @hardikpthv for the PR! amaze! :)

**Pull request merged**